### PR TITLE
Update search.md

### LINKED ISSUE #22314: How to make a SQLite column case insensitive
How to make SQLite case insensitive for column Title
- In the file *Migrations/InitialCreate.Designer.cs*, modify the column type for Title from `"TEXT"` to `"TEXT COLLATE NOCASE"`
- In the file */Pages/Movies/Index.cshtml.cs*, modify the `movies.Where` lambda to `s => EF.Functions.Like(s.Title, $"%{SearchString}%")`

Rebuild and run, the search is now case insensitive.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cc54dbf8-e6fe-a212-b2cd-c015a6ba5439
* Version Independent ID: b96b0b44-3e3a-a6c2-6b2b-c7bab4b78f0d
* Content: [Part 6, add search](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/search?view=aspnetcore-5.0)
* Content Source: [aspnetcore/tutorials/razor-pages/search.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/search.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #22314
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/search?view=aspnetcore-5.0&branch=pr-en-us-22316)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@jonpsmith please review.

File: aspnetcore/tutorials/razor-pages/search.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ If the `SearchString` property is not null or empty, the movies query is modifie
...
 
 > [!NOTE]
-> The [Contains](/dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, it's case sensitive.
+> The [Contains](/dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, it's case sensitive. See [this GitHub issue](https://github.com/dotnet/efcore/issues/11414) and [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/22314) for information on making case insensitive SQLite queries.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/search.md:65]:
```suggestion
> The [Contains](/dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, it's case sensitive. For information on making case insensitive SQLite queries, see the following:
* [This GitHub issue](https://github.com/dotnet/efcore/issues/11414).
* [This GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/22314)
* [Collations and Case Sensitivity](/ef/core/miscellaneous/collations-and-case-sensitivity)
```

**Comment by JonPSmith** [comment]
Hi @Rick-Anderson,

Summary: 
- SQL server is default setting case-insensitive
- SQLite is a mixture of case-sensitive and case-insensitive
- I would suggest you use PostgreSQL as your example of case-sensitive

## Detail

I cover collations in chapter 2 and ay [collocation unit tests](https://github.com/JonPSmith/EfCoreinAction-SecondEdition/blob/master/Test/UnitTests/TestDataLayer/Ch02_StringSearch.cs) tests all the string commands (==, Equals, StartsWith, EndsWith, Contains, IndexOf and the SQL function Like).

For SQL server the default setting case-insensitive (apart from the IndexOf because that compares a char). SQLIte by default is a mixture and I have provided the output of my unit test `TestSqliteDefault`

```text
== on SQLite is	 case-sensitive. SQL created is:
      SELECT "b"."BookId", "b"."Description", "b"."ImageUrl", "b"."Price", "b"."PublishedOn", "b"."Publisher", "b"."SoftDeleted", "b"."Title"
      FROM "Books" AS "b"
      WHERE NOT ("b"."SoftDeleted") AND ("b"."Title" = 'Entity Framework in Action')
Equals on SQLite is	 case-sensitive. SQL created is:
      SELECT "b"."BookId", "b"."Description", "b"."ImageUrl", "b"."Price", "b"."PublishedOn", "b"."Publisher", "b"."SoftDeleted", "b"."Title"
      FROM "Books" AS "b"
      WHERE NOT ("b"."SoftDeleted") AND ("b"."Title" = 'Entity Framework in Action')
StartsWith on SQLite is	 case-INsensitive. SQL created is:
      SELECT "b"."BookId", "b"."Description", "b"."ImageUrl", "b"."Price", "b"."PublishedOn", "b"."Publisher", "b"."SoftDeleted", "b"."Title"
      FROM "Books" AS "b"
      WHERE NOT ("b"."SoftDeleted") AND ("b"."Title" IS NOT NULL AND ("b"."Title" LIKE 'Entity%'))
EndsWith on SQLite is	 case-INsensitive. SQL created is:
      SELECT "b"."BookId", "b"."Description", "b"."ImageUrl", "b"."Price", "b"."PublishedOn", "b"."Publisher", "b"."SoftDeleted", "b"."Title"
      FROM "Books" AS "b"
      WHERE NOT ("b"."SoftDeleted") AND ("b"."Title" IS NOT NULL AND ("b"."Title" LIKE '%Action'))
Contains on SQLite is	 case-sensitive. SQL created is:
      SELECT "b"."BookId", "b"."Description", "b"."ImageUrl", "b"."Price", "b"."PublishedOn", "b"."Publisher", "b"."SoftDeleted", "b"."Title"
      FROM "Books" AS "b"
      WHERE NOT ("b"."SoftDeleted") AND (('Framework' = '') OR (instr("b"."Title", 'Framework') > 0))
IndexOf on SQLite is	 case-sensitive. SQL created is:
      SELECT "b"."BookId", "b"."Description", "b"."ImageUrl", "b"."Price", "b"."PublishedOn", "b"."Publisher", "b"."SoftDeleted", "b"."Title"
      FROM "Books" AS "b"
      WHERE NOT ("b"."SoftDeleted") AND ((instr("b"."Title", 'Entity') - 1) = 0)
Like on SQLite is	 case-INsensitive. SQL created is:
      SELECT "b"."BookId", "b"."Description", "b"."ImageUrl", "b"."Price", "b"."PublishedOn", "b"."Publisher", "b"."SoftDeleted", "b"."Title"
      FROM "Books" AS "b"
      WHERE NOT ("b"."SoftDeleted") AND ("b"."Title" LIKE 'Entity Framework in Action')
```

**Comment by Rick-Anderson** [comment]
> I would suggest you use PostgreSQL as your example of case-sensitive

We don't  document PostgreSQL, just SQL Sever and somewhat SQLite. Is this PR reasonable? Some folks need SQLite and it would be nice to point them off to case-insensitive information.

File: aspnetcore/tutorials/razor-pages/search.md [hunk 2]
```diff
@@ -62,7 +62,10 @@ If the `SearchString` property is not null or empty, the movies query is modifie
...
 
 > [!NOTE]
-> The [Contains](/dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, it's case sensitive.
+> The [Contains](/dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, it's case sensitive. For information on making case insensitive SQLite queries, see the following:
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/search.md:65]:
```suggestion
> The [Contains](/dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, a mixture of case sensitive and case ***IN***sensitive, depending on the query. For information on making case insensitive SQLite queries, see the following:
```

**Pull request merged**